modules/distribution/resources/api_templates/websub_api_template.xml

```diff
@@ -0,0 +1,80 @@
+<api xmlns="http://ws.apache.org/ns/synapse" name="$!apiName" context="$!apiContext" version="$!apiVersion" version-type="context">
```

Comment:
we need to have binds-t in api level as well as resource level

Reply:
Addressed in https://github.com/wso2/product-apim/pull/9885/commits/d54c96208f535aff0dacd4c3f9bcde96399d7f7d